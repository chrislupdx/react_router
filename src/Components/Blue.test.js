import React from 'react';
import { shallow } from 'enzyme';
import Blue from './Blue';

describe('Blue', () => {
  it('renders', () => {
    const wrapper = shallow(<Blue />);
    expect(wrapper).toMatchSnapshot();
  });
})
;
