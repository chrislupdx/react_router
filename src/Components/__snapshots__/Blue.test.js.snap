// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blue renders 1`] = `
<div
  style={
    Object {
      "backgroundColor": "blue",
      "height": "100vh",
      "width": "100vw",
    }
  }
/>
`;
